domain/consensus/processes/consensusstatemanager/resolve_block_status.go

```diff
@@ -9,17 +9,20 @@ import (
 )
 
 func (csm *consensusStateManager) resolveBlockStatus(blockHash *externalapi.DomainHash) (externalapi.BlockStatus, error) {
+	if blockHash.String() == "102a2c58cd2c94f1b83d1babb8aa14a5d2eb02022d6e3b9e7f58cf8659098bd9" {
+		log.Info("~~~~~~ AAAA!!! REMOVE ME!!!!")
+	}
 	// get list of all blocks in the selected parent chain that have not yet resolved their status
 	unverifiedBlocks, selectedParentStatus, err := csm.getUnverifiedChainBlocksAndSelectedParentStatus(blockHash)
 	if err != nil {
 		return 0, err
 	}
 
```

Comment:
Can you add a sanity check that `len(unverifiedBlocks) > 0`? otherwise it'll use the default value for blockStatus